<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictionarySettings</class>
 <widget class="QDialog" name="DictionarySettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Dict_path_tab">
      <attribute name="title">
       <string>Paths to dictionaries</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="path_to_dict_list">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="add_path_to_dict_button">
           <property name="text">
            <string>Add path to dictionary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="del_path_to_dict_button">
           <property name="text">
            <string>Delete path to dictionary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="del_all_paths_to_dicts_button">
           <property name="text">
            <string>Delete all paths to dictionaries</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="General_dict_group_tab">
      <attribute name="title">
       <string>Dictionary groups</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
       <item row="0" column="2">
        <widget class="QLabel" name="dict_groups_label">
         <property name="text">
          <string>Dictionary groups</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="del_all_dict_groups_button">
         <property name="text">
          <string>Delete all dictionary groups</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="avail_dicts_list"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="avail_dict_label">
         <property name="text">
          <string>Available dictionaries</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="add_dict_group_button">
         <property name="text">
          <string>Add dictionary group</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="del_dict_group_button">
         <property name="text">
          <string>Delete dictionary group</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QTextEdit" name="dict_descr_text_edit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="add_dict_to_group_button">
         <property name="text">
          <string>Add dictionary to the group</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="del_dict_from_group_button">
         <property name="text">
          <string>Delete dictionary from the group</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="show_dict_descr_button">
         <property name="text">
          <string>Show dictionary description</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QTabWidget" name="dict_groups_tab">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TTS_settings_tab">
      <attribute name="title">
       <string>Text to speech settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="tts_test_word_line_edit">
           <property name="placeholderText">
            <string>Enter a test word for TTS engine...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="tts_volume_label">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="tts_volume_slider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="tts_rate_label">
           <property name="text">
            <string>Rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="tts_rate_slider">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="tts_pitch_label">
           <property name="text">
            <string>Pitch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="tts_pitch_slider">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="tts_engine_label">
           <property name="text">
            <string>Engine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tts_engine_combobox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="tts_language_label">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tts_language_combobox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="tts_voice_name_label">
           <property name="text">
            <string>Voice name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tts_voice_name_combobox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="tts_play_button_label">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DictionarySettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DictionarySettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
